src/main/java/Duke/Main/Parser.java

```diff
@@ -0,0 +1,58 @@
+package duke.main;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import duke.errors.InvalidCommandException;
+import duke.errors.DateFormatException;
+
+/**
+ * The type Parser.
+ */
+public class Parser {
+    /**
+     * Converts a date string into LocalDateTime object
```

Comment:
Should you add punctuations behind each line?

I noticed the same issue in several other places too